templates/my.cnf.pass.erb

```diff
@@ -1,6 +1,6 @@
 [client]
 user=root
 host=localhost
-<% unless root_password == 'UNSET' -%>
-password=<%= root_password %>
+<% unless @root_password == 'UNSET' -%>
+password=<%= @root_password %>
```

Comment:
`@` is the correct sign for getting a variable from out of the scope of this file. I'm sending in another PR with more changes to the ERBs here